commerce/bookstore/python/load_data.py

```diff
@@ -0,0 +1,280 @@
+import csv, uuid, random
+import sys
+
+from typedb.client import TypeDB, SessionType, TransactionType
+import argparse
+
+# Verbosity option implementation
+parser = argparse.ArgumentParser(description='Loads data into TypeDB for the Bookstore example')
+parser.add_argument("-v", "--verbose", "-d", "--debug", help='Increase output verbosity',
+                    dest="verbosity", action='store_true')
+args = vars(parser.parse_args())
+
+if args["verbosity"]:  # if the argument was set
```

Comment:
This parameter generally gets called "verbose" - note that it's a boolean, either "true" or "false", and "verbose" is an adjective, and that it aligns with the arg name.
```suggestion
if args["verbose"]:  # if the argument was set
```
Same applies to `requests.py`

Reply:
Fixed in both files - https://github.com/vaticle/typedb-examples/pull/143/commits/57c3fa09965f67fc52fac5aad889a5f3b55b0661